src/components/Rating/Rating.st.css

```diff
@@ -0,0 +1,97 @@
+:import {
+    -st-from: "../../common/formatters.st";
+    -st-named: color, fallback;
+}
+
+:import {
+    -st-from: "../../common/palette.st.css";
+    -st-named: errorColor;
+}
+
+:vars {
+    /*Defaults*/
+    DefaultColor: color-8;
+    DefaultEmptyColor: color-3;
+    DisabledColor: color-3;
+    IconGap: 3px;
+    IconLargeGap: 11px;
+    IconLargeSize: 30px;
+}
+
+:vars {
+    /* Overrides */
+    IconColor: --overridable;
+    IconEmptyColor: --overridable;
+}
+
+.root {
+    -st-states: mobile;
+
+    overflow: hidden;
+}
+
+.root:hover .icon:not(:disabled) path {
+    stroke: color(fallback(value(IconColor), value(DefaultColor)));
+}
+
+.iconList {
+    display: flex;
+    align-items: center;
+    flex-direction: row-reverse;
+
+    float: left;
```

Comment:
We need to support `rtl` so it's better to not use floats

Reply:
But we can in `rtl` use `float: right`. It's the standard implementation with floats (even in your example from w3.org they are using floats)